brood/api.py

```diff
@@ -461,6 +462,16 @@ async def get_user_by_id_handler(
     return user
 
 
+@app.get("/auth", tags=["users"], response_model=data.UserExtendedResponse)
+async def get_auth_handler(
+    current_user: data.UserExtendedResponse = Depends(current_user_extended),
```

Comment:
Instead of `Depends`, can we make the explicit `actions.get_user_with_groups` call in handler body?

Makes it easier to modify code in future if we need to.

This is not a strong opinion, I don't mind if we don't do this.

Reply:
I afraid it will be a mess with current structure, because all sql imports in `actions` with regular `Exception` and all exceptions in `middleware.py` point to `HTTPException`